src/ert/__init__.py

```diff
@@ -26,6 +26,7 @@
     "BatchContext",
     "ErtScript",
     "JobStatus",
```

Comment:
We can not get rid of this JobStatus object here? (move it to inside batch-simulation file if it is only used there?)

Reply:
It is directly used in everest, so we will have to change it there first

Reply:
everest does an `from ert import JobStatus`, which implies that you change its location through modifying `src/ert/__init__.py`.